timescaledb/how-to-guides/update-timescaledb/index.md

```diff
@@ -1,94 +1,101 @@
-# Updating TimescaleDB versions [](update)
-
-The instructions below allow you to update TimescaleDB within the same major release
-version (for example, from TimescaleDB 2.1 to 2.2, or from 1.7 to 1.7.4). If you need
-to upgrade between TimescaleDB 1.x and 2.x, see our [separate upgrade document][update-tsdb-2]
-for detailed instructions.
-
-TimescaleDB supports **in-place updates only**: you don't need to dump and
-restore your data, and versions are published with automated migration scripts
-that convert any internal state if necessary.
-
-<highlight type="warning">
-There is currently no automated way to downgrade to an earlier release of TimescaleDB without setting up
-a new instance of PostgreSQL with a previous release of TimescaleDB and then using `pg_restore`
-from a backup.
+# Update TimescaleDB versions
+Update your TimescaleDB version with an in-place update. By updating in-place,
+you don't need to dump and restore your data. Automated migration scripts handle
+conversion of internal states where needed.
+
+<highlight type="important">
+Follow these instructions to update TimescaleDB within the same major release
+version. For example, you can update from TimescaleDB 2.1 to 2.2, or from 1.7 to
+1.7.4. To update from 1.x to 2.x, see the [section on updating to TimescaleDB 
+2.x](https://docs.timescale.com/timescaledb/latest/how-to-guides/update-timescaledb/update-timescaledb-2/).
 </highlight>
 
-### TimescaleDB release compatibility [](compatibility)
-
-TimescaleDB currently supports the following PostgreSQL releases. If you are not
-currently running a compatible release, please upgrade before updating TimescaleDB.
-
- TimescaleDB Release |   Supported PostgreSQL Release
- --------------------|-------------------------------
- 1.7                 | 9.6, 10, 11, 12
- 2.0                 | 11, 12
- 2.1-2.3             | 11, 12, 13
- 2.4                 | 12, 13
- 2.5+                | 12, 13, 14
-
-If you need to upgrade PostgreSQL first,
-see [our documentation][upgrade-pg].
-
-<highlight type="tip">
-We always recommend that you update PostgreSQL and TimescaleDB as
-separate actions to make sure that each process completes properly.
-For example, if you are currently running PostgreSQL 10 and
-TimescaleDB 1.7.5, and you want to upgrade to PostgreSQL 13 and
-TimescaleDB 2.2, upgrade in this order:
-
+## TimescaleDB release compatibility
+TimescaleDB supports the following PostgreSQL releases. If you aren't running a
+compatible release, update PostgreSQL before updating TimescaleDB. To learn
+more, see the sections on updating PostgreSQL in 
+[Timescale Cloud][cloud-update-pg], 
```

Comment:
@Loquacity I know you worked recently on something about Cloud updates, but I've trawled through the docs for the link and just can't find it. Do you remember where that section is?

Reply:
https://docs.timescale.com/cloud/latest/maintenance/#upgrade-to-a-new-postgresql-version

(From https://github.com/timescale/docs/pull/797)